### PR TITLE
add spending-prerequisite and create-card-async

### PR DESCRIPTION
**Pull request by Jianbing-immersve**
Add spending-prerequisite
Add create-card-async

Ticket Link: 
[spending-prerequisites](https://www.notion.so/immersve/Endpoint-documentation-f839590499db4836a3c7ad945abe5c1e?pvs=4) 
[create-card-async](https://www.notion.so/immersve/Endpoint-documentation-76ff28cc71d94922b7bdb9ff02eb1f75?pvs=4)

### CODE REVIEW

File: openapi/endpoints/cards/models/create-card-async-response.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+type: object
+properties:
+  cardId:
+    type: string
+    description: The created card ID
```
Comment (braden-holgate) [inline openapi/endpoints/cards/models/create-card-async-response.yaml:5]:
```suggestion
    description: The created card Id
```

File: openapi/endpoints/prerequisites/spending-prerequisites.yaml [hunk 2]
```diff
@@ -0,0 +1,106 @@
+post:
+  tags:
+    - prerequisites
+  summary: Get spending prerequisites
+  description: |
+    This endpoint returns the prerequisites required to permit the action (ex. create a new Immersve Card and load it with an amount). It also indicates if everything is in order before instructing the client of any blockchain transactions. Insufficient KYC and AML status may trigger an error response.
```
Comment (braden-holgate) [inline openapi/endpoints/prerequisites/spending-prerequisites.yaml:6]:
```suggestion
    This endpoint specifies the necessary prerequisites that must be met for a cardholder to successfully create a card and transact with it against a specified funding source. 
    The caller can specify the desired spend and receive the necessary transactions required, accounting for existing deposits, to deposit funds on-chain to meet that spend. 
    It includes any KYC and AML requirements.
```

File: openapi/endpoints/cards/create-card-async.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+post:
+  tags:
+    - card
+  summary: Create a card asynchronously
```
Comment (braden-holgate) [inline openapi/endpoints/cards/create-card-async.yaml:4]:
```suggestion
  summary: Create a card
```

File: openapi/endpoints/cards/create-card-async.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  description: |
+    Create a card under the specified funding source associated with the specified card program.
```
Comment (braden-holgate) [inline openapi/endpoints/cards/create-card-async.yaml:6]:
```suggestion
    Create a card under a specified card program and affiliated with a funding source.
```

**Review by wairowe — APPROVED** [review]

File: openapi/endpoints/cards/create-card-async.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
+post:
+  tags:
+    - card
+  summary: Create a card
+  description: |
+    Create a card under a specified card program and affiliated with a funding source.
```
Comment (rb455) [inline openapi/endpoints/cards/create-card-async.yaml:6]: "affiliated with a funding source" sounds weird to me, what about
"linked to a funding source"?

**Pull request merged**